src/main/java/se/michaelthelin/spotify/SpotifyHttpManager.java

```diff
@@ -59,6 +61,9 @@ public SpotifyHttpManager(Builder builder) {
     this.connectionRequestTimeout = builder.connectionRequestTimeout;
     this.connectTimeout = builder.connectTimeout;
     this.socketTimeout = builder.socketTimeout;
+    this.usePooledConnectionManager = builder.usePooledConnectionManager;
+    PoolingHttpClientConnectionManager connectionManagerPooling = null;
+    BasicHttpClientConnectionManager connectionManagerBasic = null;
```

Comment:
You should be able to use `HttpClientConnectionManager` to collapse both variables into one.